include/counter/helper.h

```diff
@@ -0,0 +1,10 @@
+#ifndef HELPER_H
+#define HELPER_H
+
+inline std::string get_hash_string(std::string str)
+{
+  std::size_t sz_hash = std::hash<std::string>{}(str);
```

Comment:
This is not md5, correct? Is it possible to consolidate the logic to make this the same as python layer?

Reply:
That's correct. C++ does not provide in-built md5 hashing functionality. 
We have already consolidated hashing in a common helper function "get_hash_string()".